### PR TITLE
Increases code coverage of crv_types (stats)

### PR DESCRIPTION
**Pull request by Smit-create**
Increases code coverage of `sympy.stats.crv_types`
#### References to other Issues or PRs

#18283 

#### Brief description of what is fixed or changed
Code coverage of `sympy.stats.crv_types` increased to `99.76%`

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/stats/crv.py [hunk 1]
```diff
@@ -263,13 +263,9 @@ def _moment_generating_function(self, t):
     def moment_generating_function(self, t, **kwargs):
         """ Moment generating function """
         if len(kwargs) == 0:
```
Comment (czgdp1807) [inline sympy/stats/crv.py]: What about `if not kwargs`? Which does the required by reference and not by value?

File: sympy/stats/crv_types.py [hunk 2]
```diff
@@ -190,14 +190,15 @@ def rv(symbol, cls, args):
...
     _argnames = ('a', 'b')
 
+    @property
     def set(self):
         return Interval(self.a, self.b)
 
     def pdf(self, x):
-        return 1/(pi*sqrt((x - self.a)*(self.b - x)))
+        a, b = self.a, self.b
+        return 1/(pi*sqrt((x - a)*(b - x)))
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py]: It looks like a unrelated change.

Comment (Smit-create) [reply sympy/stats/crv_types.py]: I have changed to maintain the same structure everywhere in the file

**Comment by Smit-create** [comment]
Code coverage increases. I think it is good to go.

**Comment by czgdp1807** [comment]
LGTM. Will merge if no one objects for some time.
